packs/sonarqube/README.md

```diff
@@ -0,0 +1,73 @@
+# sonarqube-nomad-pack
+
+This pack contains a service job that runs Sonarqube in Nomad. It currently supports being run by the [Docker](https://www.nomadproject.io/docs/drivers/docker) driver.
+
+## Requirements
+
+Clients that expect to run this job require:
+- [Docker volumes](https://www.nomadproject.io/docs/drivers/docker "Docker volumes") to be enabled within their Docker plugin stanza, due to the usage of Nomad's host volume:
+```hcl
+plugin "docker" {
+  config {
+    volumes {
+      enabled = true
+    }
+  }
+}
+```
+
+- [Host volume](https://www.nomadproject.io/docs/configuration/client#host_volume-stanza "Host volume") to be enabled in the client configuration (the host volume directory - /opt/sonarqube/data - must be created in advance):
+```hcl
+client {
+  host_volume "sonarqube" {
+    path      = "/opt/sonarqube/data"
+    read_only = false
+  }
+}
+```
+
+- If you're running on Linux, you must ensure to set these values by running the following commands as root:
```

Comment:
Would you be able to expand on why we need these values so future readers can get context on why they are needed?